index.html

```diff
@@ -13,6 +13,50 @@
   </head>
 
   <body>
-   
+    <div class="container">
+      <header>
+        <div class="header">
+          <div class="header-p">
+            <p class="name">Kevin Toro</p>
+          </div>
+          <div class="fa-bars-icon">
+            <i class="fa-solid fa-bars"></i>
+          </div>
+        </div>
+      </header>
+      <main>
+        <section class="section-1">
+          <div class="headline">
+            <div class="Primary-text">Hey There. I’m Toro</div>
+            <div class="Supporting-text">I’m a Software Developer</div>
+            <div class="Description">
+              I can help you build a product , feature or website Look through
+              some of my work and experience! If you like what you see and have
+              a project you need coded, don’t hestiate to contact me.
+            </div>
+            <div class="social-media">
+              <img
+                src="/portfolio-first-project/images/github.png"
+                alt="github-icon"
+              />
+              <img
+                src="/portfolio-first-project/images/twitter.png"
+                alt="twitter-icon"
+              />
+              <img
+                src="/portfolio-first-project/images/linkedIn.png"
+                alt="linkedIn-icon"
+              />
+              <img
+                src="/portfolio-first-project/images/angel list.png"
+                alt="chill-icon"
+              />
+              <img src="/portfolio-first-project/images/M.png" alt="M-icon" />
+            </div>
```

Comment:
In this scenario, we can simply consider it as a list of social media icons, and list type is unordered one. So we will be using `ul` tag instead of div. And inside this `ul` tag there would be `li` tags. And each `li` tag would be in an `a` tag. containing one social media icon in it. So:
`ul > li > a > img`